Gopkg.lock

```diff
@@ -211,50 +195,6 @@
   revision = "792786c7400a136282c1664665ae0a8db921c6c2"
   version = "v1.0.0"
 
-[[projects]]
-  digest = "1:a22debd993d18b9203c985810f33f52e7583724800ee8d15e7d362e7175639d1"
-  name = "github.com/prometheus/client_golang"
```

Comment:
Looks like this is removing the prometheus dependency ?

Reply:
@Geod24 Yes, it is not used at this time. Did I miss something?

Reply:
We did. https://github.com/bosnet/sebak/commit/f776bf0066f3e8191e18cf14c28321d60538f45c#diff-bc5b5719c421f802a296dffd1525b2f4L180
This line shouldn't have been removed.

Reply:
@Geod24 Hmm, it was removed. I see. We can recover it later, right?

Reply:
I think we should recover it now. Doesn't make sense to remove this dependency to re-add it in a few commits.

Reply:
I'll raise a PR.

Reply:
@Geod24 This PR should not keep the useless one and the unused packages in Gopkg also should not be in here.Anyway it's not matter to keep the old one.